subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -541,6 +541,11 @@ static int __wifi_args_to_params(size_t argc, char *argv[],
 		idx++;
```

Comment:
Please change the commit subject to something like
`net: wifi: Add channel validity check for AP mode`